patterns/atoms/media/video/video.twig

```diff
@@ -0,0 +1,4 @@
+<video width="{{video.width}}" height="{{video.height}}" controls="{{video.controls}}" class="{{video.class}}">
+	<source src="{{video.src}}" type="video/ogg">
```

Comment:
we should have a loop for variable here like {{sources}} and render `<source>` tags inside to make it more flexible